css/main.css

```diff
@@ -27,4 +27,3 @@ body {
   height: 27px;
 
   text-align: right;
-}
```

Comment:
Whoops! Sorry about that. Have to work on my code cleanliness. This is my first attempt at contribution and I'm pretty nervous about it.